lang/java/reef-tang/tang/src/main/java/org/apache/reef/tang/implementation/Subplan.java

```diff
@@ -23,9 +23,10 @@
 import java.util.*;
 
 public final class Subplan<T> extends InjectionPlan<T> {
-  final InjectionPlan<? extends T>[] alternatives;
-  final int numAlternatives;
-  final int selectedIndex;
+
+  private final InjectionPlan<? extends T>[] alternatives;
+  private final int numAlternatives;
+  private final int selectedIndex;
```

Comment:
I asked these visibility-change (`private`) PR. It seems that I wasn't clear enough.

Reply:
BTW, never mind any more about that.